addons/web/static/src/views/fields/properties/property_value.js

```diff
@@ -149,6 +149,10 @@ export class PropertyValue extends Component {
 
         if (this.props.type === "many2one" && value && value.length === 2) {
             return formatMany2one(value);
+        } else if (this.props.type === "integer") {
+            return formatInteger(value);
+        } else if (this.props.type === "float") {
+            return formatFloat(value);
```

Comment:
You can restore this diff, I don't see the importance. But in general, I'm okay with this PR.

Reply:
Oops. Okay. My bad.